README.md

```diff
@@ -58,13 +58,15 @@ class Robot:
 
 We intend to support the kits of the following robotics competitions:
 
-- [SourceBots](https://sourcebots.co.uk/)
+- [SourceBots Summer School](https://sourcebots.co.uk/)
 - [Student Robotics](https://studentrobotics.org/)
 - [Hills Road RoboCon](https://hr-robocon.org/)
 
 Whilst `j5` isn't officially endorsed by Student Robotics or RoboCon, we are working closely with Student Robotics to ensure perfect compatibility. Many `j5` contributors are members of Student Robotics and SourceBots.
 
-If you are interested in adding support for your hardware, please get in touch.
+[sbot](https://github.com/sourcebots/sbot), a `j5` based API, was successfully deployed to over 100 users in August 2019. This is the first known case of a real-world deployment of a `j5` based API.
```

Comment:
I thought we had 70 competitors? Or are you including volunteers in this figure too?

Reply:
I'm including 70 competitors, 20ish volunteers and 10ish smallpeice staff. Emphasis on _user_.